tests/foreman/cli/test_reporttemplates.py

```diff
@@ -693,3 +694,70 @@ def test_positive_generate_with_name_and_org(self):
             host['name'],
             [item.split(',')[0] for item in result]
         )
+
+    @tier2
+    @pytest.mark.skip_if_open('BZ:1782807')
+    def test_positive_generate_ansible_template(self):
+        """ Report template named 'Ansible Inventory' (default name is specified in settings)
+        must be present in Satellite 6.7 and later in order to provide enhanced functionality
+        for Ansible Tower inventory synchronization with Satellite.
+
+        :id: f1f7adfc-9601-4498-95c8-3e82e2b36583
+
+        :setup:
+            1. A user with minimal required permissions: 'Ansible Tower Inventory Reader' role
+            2. A fake host to be checked in report output
+
+        :steps:
+            1. Check settings for default Ansible Inventory template name and ensure
+               the template is present
+            2. Try to render the template using the user with ATIR role
+            3. Check the fake host is present in the output
+
+        :expected result: Report template is present, renederable and provides output
```

Comment:
It's `:expectedresults:` - without space and in plural. Right now it causes travis failure.

Reply:
Thank you for pointing out, I have overlooked this.